blog/src/__tests__/Footer.test.js

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import Footer from '../components/Footer/Footer';
+
+describe('Unit tests for Footer Component', () => {
```

Comment:
I think it would be a check if proper semantic tags are also used. For example, footer tag should wrap the footer.